gordo/builder/build_model.py

```diff
@@ -287,6 +288,7 @@ def _build(self) -> Tuple[sklearn.base.BaseEstimator, Machine]:
                 model_meta=self._extract_metadata_from_model(model),
             )
         )
+        machine.report()
```

Comment:
This just rubs me the wrong way. I don't know why really. It is just a bit weird that a machine reports itself. I dont know what the alternatives are, and if they are nicer:
- Have the for loop here in build_model (maybe as a function),
- Have the for loop in `gordo/reporters/base.py `, maybe as a class-function?